mods/d2k/rules/world.yaml

```diff
@@ -35,6 +35,10 @@
 		Name: Corrino
 		InternalName: corrino
 		Selectable: false
+	Faction@Smuggler:
```

Comment:
IMO it would make sense to have a dedicated PR for setting up the smuggers/mercenaries(/fremen?) factions properly.